common/lib/xmodule/xmodule/util/adaptive_learning.py

```diff
@@ -222,8 +237,14 @@ def create_knowledge_node_student(self, knowledge_node_uid, student_uid):
         """
         url = self.knowledge_node_students_url
         payload = {'knowledge_node_uid': knowledge_node_uid, 'student_uid': student_uid}
-        response = requests.post(url, headers=self.request_headers, data=payload)
-        knowledge_node_student = json.loads(response.content)
+        try:
+            response = requests.post(url, headers=self.request_headers, data=payload)
+            knowledge_node_student = json.loads(response.content)
+        except requests.ConnectionError:
+            knowledge_node_student = {}
+            log.warning("Could not create 'knowledge node student' object "
+                        "that links student identified by student_uid {student_uid} "
+                        "to unit identified by knowledge_node_uid {knowledge_node_uid}.".format(student_uid=student_uid, knowledge_node_uid=knowledge_node_uid))
```

Comment:
@macornwell This line is still too long. [Line length should not exceed 120 chars.](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#syntax-and-organization)